imports/ui/components/modals/editProfileInfo.html

```diff
@@ -0,0 +1,35 @@
+<template name="editProfileInfo">
```

Comment:
@jrgarou I refactored this modal to try and conform it to the standard you have established. Let me know if this was the right direction/what might need changing!

Reply:
Great! But I don't know if we have to work this in a modal context.
@pedrocasa and I wants to discuss this, because I think the menu has to be in the button context. 
In the video list page there's a "watch later/add to playlist" menu, and I believe this menu must be in thumb context and not set a position for every one.